src/room.ts

```diff
@@ -10,16 +10,17 @@ import { Message }    from './message'
 import { UtilLib }    from './util-lib'
 
 type RoomObj = {
-  id:         string,
-  encryId:    string,
-  topic:      string,
-  ownerUin:   number,
-  memberList: Contact[],
-  nameMap:    Map<string, string>,
-  aliasMap:   Map<string, string>,
+  id:               string,
+  encryId:          string,
+  topic:            string,
+  ownerUin:         number,
+  memberList:       Contact[],
+  nameMap:          Map<string, string>,
+  roomAliasMap:     Map<string, string>,
+  contactAliasMap:  Map<string, string>,
 }
 
-type NameType = 'name' | 'alias'
+type NameType = 'name' | 'roomAlias' | 'contactAlias'
```

Comment:
API should never be break without a major version change.  

Know more about SemVer, Please read from here: http://semver.org/

So you have to keep the `alias` for the old version API users.

I will suggest to use `alias` as an alias of `roomAlias`, because the Class is `Room`.